source/guides/network/supernode-enrollment.rst

```diff
@@ -84,54 +94,86 @@ After following that guide, you need to **install a monitoring agent** and **ann
-      openssl x509 -req -days 999 -in agent-csr.pem -out agent-crt.pem -signkey agent-key.pem
+   - Create a new file named ``agent-ca.cnf`` with the following content:
 
-   You also need to download the **Symbol network CA certificate**. This allows the agent to authenticate connections to the node it is monitoring.
+     .. code-block:: ini
 
-   .. code-block:: bash
+        [ca]
+        default_ca = CA_default
+        [CA_default]
+        policy = policy_catapult
+        [policy_catapult]
+        commonName = supplied
+        [req]
+        prompt = no
+        distinguished_name = dn
+        [dn]
+        CN = Agent CA
+   
+   - Then generate the certificate by running:
+
+     .. code-block:: bash
+
+        # Generate agent CA key pair
+        openssl genpkey -out agent-ca.key.pem -outform PEM -algorithm ed25519
+
+        # Create agent CA CSR
+        openssl req -config agent-ca.cnf -key agent-ca.key.pem -new -out agent-ca.csr.pem
+
+        # Base64 encode agent CA CSR file for later use
+        base64 agent-ca.csr.pem --wrap 0 ; echo
+
+     The last command outputs a string of characters. Copy them to be used later.
+
+   - Finally, download the Symbol network CA certificate to allow the agent to authenticate connections to the node it is monitoring (Download only one):
 
-      wget https://symbol-node-reward.s3-eu-west-1.amazonaws.com/mainnet/certs/ca-crt.pem
+     .. code-block:: bash
+
+        # For MAINNET
+        wget https://symbol-node-reward.s3-eu-west-1.amazonaws.com/mainnet/certs/controller-ca-crt.pem
+
+        # For TESTNET
+        wget https://symbol-node-reward.s3-eu-west-1.amazonaws.com/testnet/certs/controller-ca-crt.pem
+
+   - Move out of the ``certs`` folder.
 
 3. **Configure the monitoring agent**:
 
    Create a text file named ``agent.properties`` and add the content below, replacing ``NODE_PRIVATE_KEY`` and ``REST_GATEWAY_URL`` with the appropriate values:
 
    .. code-block:: properties
 
-      NETWORK_TYPE=104 ; 104 for MAINNET, 152 for TESTNET
-      ; Node's transport private key
-      NODE_PRIVATE_KEY=●●●●●●●●●●●●●●●●●●●●●●●●●●●●●●●●●●●●●●●●●●●●●●●●●●●●●●●●●●●●●●●●
+      ; Use 104 for MAINNET or 152 for TESTNET
+      NETWORK_TYPE=104 
       LOGGER_FILE=agent.log
       ; Replace with the public host where your node is running (hostname or IP address)
       REST_GATEWAY_URL=http://my-symbol-node.com:3000
       REWARD_PROGRAM=SuperNode
-      CONTROLLER_PUBLIC_KEY=645FE86EAED04747DE066D838FE6E77BA2B146DC8BBC505617323D5DF01106A5 ; MAINNET
-      ; CONTROLLER_PUBLIC_KEY=68B6A1D2F292E75F9BB8E9EDDA086A7C293A198C9968FF7528374075AAF4D983 ; TESTNET
-      CERTS_CA_FILE=ca-crt.pem
-      CERTS_KEY_FILE=agent-key.pem
-      CERTS_CERT_FILE=agent-crt.pem
+      ; Replace with the main account's public key
+      MAIN_PUBLIC_KEY=●●●●●●●●●●●●●●●●●●●●●●●●●●●●●●●●●●●●●●●●●●●●●●●●●●●●●●●●●●●●
+      CERTS_CONTROLLER_CA_CERT_FILE=./certs/controller-ca.crt.pem
```

Comment:
Can we replace `controller-ca.crt.pem` with `controller-ca-crt.pem` in the file path? Going with the hyphen in the S3 bucket as well.

Reply:
Oops! typo.